src/ucp/api/ucp.h

```diff
@@ -3355,6 +3355,17 @@ ucp_atomic_fetch_nb(ucp_ep_h ep, ucp_atomic_fetch_op_t opcode,
  *          Therefore @a param->reply_buffer and its corresponding flag
  *          @ref UCP_OP_ATTR_FIELD_REPLY_BUFFER have to be defined in @a param.
  *
+ * <table>
+ * <caption>Mandatory fields in ucp_request_param_t</caption>
+ * <tr> <td align="center">Atomic Operation</td> <td align="center">Fields</td>
```

Comment:
pls specify for each operation where is the argument for the operation and where the result is placed.
for cswap, there are to arguments: compare-arg in "buffer" and swap-arg in "replyu_buffer"